composer.json

```diff
@@ -80,16 +80,21 @@
         "egulias/email-validator": "~1.2"
     },
     "autoload": {
-        "psr-0": { "Symfony\\": "src/" },
+        "psr-0": {
+            "Symfony\\Bridge\\Doctrine\\": "src/",
+            "Symfony\\Bridge\\Monolog\\": "src/",
+            "Symfony\\Bridge\\ProxyManager\\": "src/",
+            "Symfony\\Bridge\\Swiftmailer\\": "src/",
+            "Symfony\\Bridge\\Twig\\": "src/",
+            "Symfony\\Bundle\\": "src/",
+            "Symfony\\Component\\": "src/"
```

Comment:
Is this really an issue at all given that you can dump an optimized autoloader (the generated class map will not differ)?